sql/core/src/main/scala/org/apache/spark/sql/jdbc/README.md

```diff
@@ -46,6 +46,12 @@ so they can be turned off and can be replaced with custom implementation. All CP
 which must be unique. One can set the following configuration entry in `SparkConf` to turn off CPs:
```

Comment:
Actually, this option has to be placed in `StaticSQLConf`, or should be effective in runtime. I will make a separate PR to move it for now.

Reply:
https://github.com/apache/spark/pull/33381

Reply:
I will rebase, thanks.